src/query/storages/fuse/fuse/src/fuse_table.rs

```diff
@@ -388,12 +424,14 @@ impl Table for FuseTable {
         overwrite: bool,
     ) -> Result<()> {
         self.check_mutable()?;
+        let ndvs = calc_column_ndvs_of_data_blocks(&operations)?;
```

Comment:
the data blocks referred to by `operations` are `AppendOperationLogEntry`s, not the data blocks appended to table